menu_technical_info/static/src/js/menu_technical_info.js

```diff
@@ -0,0 +1,33 @@
+odoo.define('menu_technical_info.Menu', function (require) {
```

Comment:
I think this is a matter of style. If not, can you please send me your eslint configuration.